### ISSUE TITLE
Unit testing

### ISSUE THREAD

**Issue by BryceFury** [BryceFury/tumbly#4]
As title.

**Comment by pratyushprakash** [comment]
Hi, can I take this up?

**Comment by BryceFury** [comment]
Of course, ignore run.py for now as I'm just finishing up the new GUI.

**Comment by pratyushprakash** [comment]
How would you like me to go about testing?

**Comment by pratyushprakash** [comment]
do you want me to write a separate setup file for the imports or shift the tests to tumbly/tumbly

**Comment by BryceFury** [comment]
Go for separate, that way tumbly/tumbly can only contain things needed for the program to actually run as I'd like to freeze it to make an app/exe in future and the tests won't be necessary for that.
